code/datums/mood_events/generic_negative_events.dm

```diff
@@ -310,3 +310,8 @@
 	description = "<span class='warning'>All the fish are dead...</span>\n"
 	mood_change = -3
 	timeout = 90 SECONDS
+
+/datum/mood_event/burnt_wings
+	description = "<span class='boldwarning'>My wings were RUINED!!</span>\n"
```

Comment:
I don't like the wording on this, "were" is definitely not correct. Here's my recommendation based on the in game text, feel free to play with it:

```suggestion
	description = "<span class='boldwarning'>MY PRECIOUS WINGS!!</span>\n"
```

Reply:
Were was thinking about how there's nothing stopping it from happening and the situation changing, same as if someone gets an arm severed and then replaced with a prosthetic. 

I'm in no way invested in the wording though.